examples/tutorials/Druggablity_assessment_with_Fpocket.ipynb

```diff
@@ -0,0 +1,2112 @@
+{
```

Comment:
<p>The sentence effectively simplifies the concept for a general audience, which is great. However, I find that the (especially from "Think of it ... sites.") clarity could be improved slightly. I don't know the topic well enough to suggest an alternative.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>